src/AutoRest.CSharp/Mgmt/AutoRest/MgmtConfiguration.cs

```diff
@@ -350,6 +350,7 @@ internal void SaveConfiguration(Utf8JsonWriter writer)
             WriteNonEmptySettings(writer, nameof(PrivilegedOperations), PrivilegedOperations);
             WriteNonEmptySettings(writer, nameof(OverrideOperationName), OverrideOperationName);
             WriteNonEmptySettings(writer, nameof(PartialResources), PartialResources);
+            WriteNonEmptySettings(writer, nameof(RawParameterizedScopes), RawParameterizedScopes);
```

Comment:
we need to write this into the json in order to properly debug projects with this configuration set.